README.md

```diff
@@ -43,3 +43,134 @@ Always use a GPL-compatible license.
 
 Perhaps the best-known license is the GNU Public License (GPL), which guarantees the freedom of users to use, copy, and modify code. Code released under a GPL-compatible license is code that can be incorporated into another GPL-licensed codebase without modification to the license. [Reference](https://www.astrobetter.com/blog/2014/03/10/the-whys-and-hows-of-licensing-scientific-code/)
 
+
+# HTML helpful Information 
```

Comment:
can you instead create read me in a HTML folder and add info on this read me

Reply:
Changes has been done as per review